src/cli/fmt.rs

```diff
@@ -40,6 +44,23 @@ impl Fmt {
     table.push('\n');
     table
   }
+  pub fn format_operation(op: &str, errors: &[async_graphql::ServerError]) -> String {
+    let meta = if errors.is_empty() {
+      Fmt::success(&"No errors found".to_string())
+    } else {
+      Fmt::meta(&format!("{} errors found", errors.len()))
+    };
+    let heading = Fmt::heading(&format!("Operation {op}: {meta}"));
+
+    let report = errors
+      .iter()
+      .enumerate()
+      .map(|(count, error)| Fmt::error(&format!("{}. {}", count + 1, error.message)))
+      .collect::<Vec<String>>()
+      .join("\n");
+
+    format!("{heading}\n{report}")
```

Comment:
If you return a `Valid` it can be easily handled by the CLI internally because it's built on a ValidationError which is converted to a `CLIError` which is CLI friendly.